thirdeye-ui/src/app/components/alert-list-v1/alert-list-v1.component.tsx

```diff
@@ -223,7 +222,6 @@ export const AlertListV1: FunctionComponent<AlertListV1Props> = ({
                             order: DataGridSortOrderV1.DESC,
                         }}
                         rowKey="id"
-                        scroll={DataGridScrollV1.Body}
                         searchPlaceholder={t("label.search-entity", {
```

Comment:
there is another scroll option from DataGridScrollV1 maybe use that?

Reply:
Didn't understand this - "scroll will now show last entries ".
Could you please help me with a SS of this